packages/espressocash_app/lib/features/ramp/partners/moneygram/service/moneygram_on_ramp_service.dart

```diff
@@ -283,9 +283,12 @@ class MoneygramOnRampOrderService implements Disposable {
       );
     }
 
-    final amount = CryptoAmount(
-      value: order.amount,
-      cryptoCurrency: Currency.usdc,
+    final amount =
+        Amount.fiat(value: order.amount, fiatCurrency: Currency.usd).let(
```

Comment:
I don't get it, why does it need to be converted to fiat first?

Reply:
`order.amount` is int value from fiat amount, and bridge needs precision in USDC